CMakeLists.txt

```diff
@@ -225,6 +225,16 @@ set(REQUIRED_LIBS
   zlib
 )
 
+option(usebpf "Enable bpf acceleration")
+
+if(usebpf)
+  add_definitions(-DUSEBPF=1)
```

Comment:
Bikeshed stuff, but I think `usebpf`/`USEBPF` are unnecessarily hard to read. I'd prefer to go with just `bpf`/`BPF` or possibly `enable_bpf`/`ENABLE_BPF`.